src/core/server/config/__tests__/__mocks__/env.ts

```diff
@@ -21,33 +21,14 @@
 
 import { EnvOptions } from '../../env';
 
-interface MockEnvOptions {
-  config?: string;
-  kbnServer?: any;
-  mode?: EnvOptions['mode']['name'];
-  packageInfo?: Partial<EnvOptions['packageInfo']>;
-}
-
-export function getEnvOptions({
-  config,
-  kbnServer,
-  mode = 'development',
-  packageInfo = {},
-}: MockEnvOptions = {}): EnvOptions {
+export function getEnvOptions(options: Partial<EnvOptions> = {}): EnvOptions {
   return {
-    config,
-    kbnServer,
-    mode: {
-      dev: mode === 'development',
-      name: mode,
-      prod: mode === 'production',
-    },
-    packageInfo: {
-      branch: 'some-branch',
-      buildNum: 1,
-      buildSha: 'some-sha-256',
-      version: 'some-version',
-      ...packageInfo,
+    configs: options.configs || [],
```

Comment:
__note:__ Kibana can rely on multiple configs at the same time (content will be merged).